client/src/lib/localStorage.js

```diff
@@ -0,0 +1,11 @@
+export const loadFromLocal = (key) => {
+  try {
+    const local = JSON.parse(localStorage.getItem(key));
```

Comment:
You could consider summarizing both lines in one: `return JSON.parse(localStorage.getItem(key));`

Reply:
see comment below :)